packages/vscode-extension/src/webview/components/DeviceSettingsDropdown.tsx

```diff
@@ -153,6 +154,24 @@ function DeviceSettingsDropdown({ children, disabled }: DeviceSettingsDropdownPr
               </DropdownMenu.SubContent>
             </DropdownMenu.Portal>
           </DropdownMenu.Sub>
+          <Label>Screen recording</Label>
+          <div className="dropdown-menu-item">
+            <span className="icons-container">
+              <span className="codicon codicon-triangle-left icons-rewind" />
+              <span className="codicon codicon-triangle-left icons-rewind" />
+            </span>
+            Enable Replays
+            <Switch.Root
+              className="switch-root small-switch"
+              id="enable-replays"
+              onCheckedChange={(checked) =>
+                project.updateDeviceSettings({ ...deviceSettings, replaysEnabled: checked })
```

Comment:
I belive this option should be part of workspace settings (we have a separate interface for that called `WorkspaceConfig` as it is not a device setting also it would be nice if this option synchronized between computers for users that are logged in to microsoft account

Reply:
I thought about it, but feel like it fits the device setting pane better for now. 

The workspace settings doesn't have any configuration options except from the IDE panel location for now. Adding it there would make it an isolated option. For me it still feels like its device related because we're recording the device screen in the end. The only thing is that it affects the panel UI as it adds a new button when replays are enabled.

I'd keep it as is for now, and we can consider moving it in the future or once we have more similar options added that'd make it less of a precedence to have a switch button included in the workspace settings dropdown.

Reply:
I do not agree, for the following reasons: 
- `WorkspaceConfig` currently includes showDeviceFrame setting that is affecting user ui in a similar way as replay button does, (it is not included in the settings dropdown but it is accessible through vscode settings panel) 
- it does not change anything on the device were as all other device settings do, I see your point that it records device but the recording does not happen on the device itself which means that it won't affect how device works during testing. As a user I would see the device setting menu as a menu i use often while testing my application and all other options as dose i set up only when something goes wrong. 
- this is a setting that is rarely used (probably only once per user) as a User does not have much reason to use it except their preference  which I expect does not change often
- workspace settings persist across devices for logged in user which is a useful functionality that we get for free 
- I'm not even sure if this setting needs to exist in the UI because of how rarely it should be used and it's location in the vscode settings would make it easy to remove it from UI settings if they get to cluttered